commander/test/utils/reader.spec.ts

```diff
@@ -162,5 +164,50 @@ describe('reader', () => {
 				return expect(result).toEqual(['passphrase', 'password', 'data']);
 			});
 		});
+
+		describe('getFileExtension', () => {
+			it('should return a json file extension', () => {
+				const filePath = './some/path.json';
+				const fileExtension = getFileExtension(filePath);
+
+				return expect(fileExtension).toEqual('.json');
+			});
+
+			it('should throw an error if no file extension is passed', () => {
+				const filePath = './some/path';
+				expect(() => getFileExtension(filePath)).toThrow('Not a JSON file.');
+			});
+		});
+
+		describe('readParamsFile', () => {
```

Comment:
maybe also add a test case for failure i.e. `FILE_NOT_FOUND`

Reply:
Done!